ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/commands/SearchFormsLoaderCommand.java

```diff
@@ -87,7 +89,9 @@ protected Object executeWithSubject() throws Exception {
     console.println(ansi().fgBrightCyan().a("Initializing Search Form Template Loader").reset());
     final SearchFormsLoader loader = generateLoader();
 
-    List<Metacard> parsedMetacards = loader.retrieveSystemTemplateMetacards();
+    List<Metacard> parsedMetacards =
+        AccessController.doPrivileged(
+            (PrivilegedAction<List<Metacard>>) () -> loader.retrieveSystemTemplateMetacards());
```

Comment:
❓ Interesting - we didn't need to do this previously. Did the DDF UI separation influence this?

Reply:
Hmm I think you are right. I see these file permissions in the default policy https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/security/default.policy#L393-L396

Reply:
I'm not too familiar with how the policies are enforced. Do you think I would need to create a new policy for ddf-ui to install with those permissions?

Reply:
No we're good 👍